CONTRIBUTING.rst

```diff
@@ -29,86 +29,80 @@ Documentation Convention
 
 Our documentation project setup can be found in the `README. <https://github.com/globalwordnet/gwadoc/blob/master/README.md>`_
 
-Start reading our documentation and you’ll get the hang of it. We optimize for readability.
-We indent using two spaces.
+Most of our contributions happen in the ``doc_en.py`` file and we generate the html files from this automatically.
 
-Most of our contributions happen in the ``doc_en.py`` file and we
-generate the html files from this automatically.
+When writing please consider your readers, we would like the documentation reader friendly for beginners as well as useful for experts.
 
-When writing please consider your readers, we would like the
-documentation reader friendly for beginners as well as useful for
-experts.
+How to add a new relation
+=========================
+
+Documentation for a new relation should be done after discussing the relation with generally.
```

Comment:
I'm not sure exactly what you were going for here. Maybe "after discussing the relation with the group"?

Reply:
Corrected